app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/profile/view/ProfileStatsFragment.kt

```diff
@@ -58,7 +59,7 @@ class ProfileStatsFragment : Fragment() {
         profileResponse?.asStudent?.let {
             val asStudentDetails: AsStudentDetails = it
             val text = "Total impact made by $username's students"
```

Comment:
Can this hardcode text be moved to the `strings.xml` ? Once every text is removed, the Suppress annotation above can be removed.

Reply:
Thanks for your observation. I will work on it.